### ISSUE TITLE
String is clickable

### ISSUE THREAD

**Issue by Hativ3** [AsaAyers/js-hyperclick#64]
A string of the name like a variable is clickable.

Example:
> var foo;
> 
> var obj = {
>     "foo": "bar"
> };

`foo` in `obj` will be clickable and jump to `var foo;`

**Comment by AsaAyers** [comment]
I don't think this is something I want to address with the current design. When you over over `"foo"` in the object `hyperclick` provides the string and the physical location. `js-hyperclick` just looks up the known variables to see if there is a match.

I guess to solve this I would need to locate every place a variable is used and store those locations. I don't think the current behavior causes a problem, so this is a low priority.
